server/data/testData/FINDS.ts

```diff
@@ -0,0 +1,5351 @@
+      label: 'WERE THE ITEMS OBTAINED ON TEMPORARY RELEASE?',
+      multipleAnswers: false,
+      answers: [
+        {
+          id: '219123',
+          code: 'NO',
+          active: true,
+          label: 'NO',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219124',
+          code: 'UNKNOWN',
+          active: true,
+          label: 'UNKNOWN',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219125',
+          code: 'YES',
+          active: true,
+          label: 'YES',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: '67227',
+        },
+      ],
+    },
+    '67227': {
+      id: '67227',
+      active: true,
+      code: 'DESCRIBE THE TYPE OF TEMPORARY RELEASE',
+      label: 'DESCRIBE THE TYPE OF TEMPORARY RELEASE',
+      multipleAnswers: true,
+      answers: [
+        {
+          id: '219126',
+          code: 'COMMUNITY VISIT',
+          active: true,
+          label: 'COMMUNITY VISIT',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219127',
+          code: 'COMPASSIONATE',
+          active: true,
+          label: 'COMPASSIONATE',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219128',
+          code: 'FACILITY',
+          active: true,
+          label: 'FACILITY',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219129',
+          code: 'RESETTLEMENT',
+          active: true,
+          label: 'RESETTLEMENT',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+        {
+          id: '219130',
+          code: 'UNKNOWN',
+          active: true,
+          label: 'UNKNOWN',
+          commentRequired: false,
+          dateRequired: false,
+          nextQuestionId: null,
+        },
+      ],
+    },
+  },
+  prisonerRoles: [
+    {
+      prisonerRole: 'ASSISTED_STAFF',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'IMPEDED_STAFF',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'IN_POSSESSION',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'INTENDED_RECIPIENT',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'PRESENT_AT_SCENE',
+      onlyOneAllowed: false,
+      active: true,
+    },
+    {
+      prisonerRole: 'SUSPECTED_INVOLVED',
+      onlyOneAllowed: false,
+      active: true,
+    },
+  ],
+} as const
+
+export const steps: FormWizard.Steps = {
```

Comment:
interesting chicken and egg… was this generated?

Reply:
yeah, this is the generated config that I have moved under testData so that it doesn't change for whatever reason affecting the test.